sql/src/test/java/io/crate/execution/engine/window/BasicWindowFunctionTest.java

```diff
@@ -44,4 +47,22 @@ public void testInputOfDifferentSizeRaiseException() throws Exception {
             new Object[]{1, 2}
         );
     }
+
+    @Test
+    public void testAggregationFunctionsAreNotSupported() throws Exception {
```

Comment:
This would fail if assertions are disabled... I also don't see a need to test for assertions

Reply:
removed